src/main/java/seedu/address/logic/descriptors/InvalidFilterApplicantDescriptor.java

```diff
@@ -0,0 +1,24 @@
+package seedu.address.logic.descriptors;
+
+import static seedu.address.model.applicant.ApplicantMatchesFiltersPredicate.passesFilter;
+
+import seedu.address.model.Model;
+
+/**
+ * A descriptor used to verify the validity of a FilterApplicantDescriptor.
+ */
+public class InvalidFilterApplicantDescriptor extends FilterApplicantDescriptor {
```

Comment:
```suggestion
public class FilterApplicantDescriptorVerifier extends FilterApplicantDescriptor {
```
Invalid sounds a little strange, but I get where you're coming from.

Reply:
Thanks! That does sound better.